src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -706,6 +706,12 @@ public CheckContextSummary ApplyGetContextSummary()
 
             return summary;
         }
+
+        public IEnumerable<string> GetFunctionNames(bool fullNames = true)

```

Comment:
>bool fullNames = true [](http://example.com/codeflow?start=52&length=21)

Is there a scenario where we **don't** want the full names? If so, can we avoid this argument?

Reply:
There are other hosts that can use the short name option.